spring-xd-use-cases.tex

```diff
@@ -1,2 +1,11 @@
 \section{Use Cases}
-This section describes use cases for Spring XD.
+Few Fortune 100 and several Fortune 1000 companies are using Spring XD but all are under NDA. Following are some of the common use-cases that we have been supporting in the field.
+
+\begin{itemize*}
+\item \textbf{Fault Detection}: Spring XD connects machine data with enterprise infrastructure, middleware, and backend services. Continuous streams can be orchestrated in Spring XD to connect with various devices (i.e., machines). Consume and transform data of varied formats, analyze, predict failures, generate reports, and dispatch maintenance personnel - all in real-time.
+\item \textbf{Enterprise Modernization}: Enterprises invest heavily on IT infrastructure, as the data resides at several layers in the enterprise architecture. Tight coupling with various products further adds more overhead with data management. Spring XD, as one-stop runtime, provides data integration adapters to consume data of varied formats (i.e., structures, unstructured, binary, ..) that reside in different toolchains (i.e., database, middlewares, in-memory grids, ..). Given the unified approach, enterprises' are equipped with developer-friendly fixtures to create pipelines using Spring XD. Pipelines connecting varied data producing and consuming agents eliminates the necessity of toolchains, maintenance overhead, or production support - Spring XD simplifies data collection and aggregation.
+\item \textbf{Data Ingest}: Spring XD is a standard tool for ingesting data into Hadoop. Whether it is real-time (i.e., online) or batch (i.e., offline), you've fixtures to operationalize pipeline that fits your business needs.
+\item \textbf{24/7 Production Pipelines}: The business demands highly available data streams with guaranteed data processing, to react in real-time accurately. Spring XD's runtime is highly reliable that can recover from failures seamlessly. Production running pipelines are long running tasks - there's no end. For ad-hoc operations such as querying, machine learning, or data crunching, `taps' in Spring XD are commonly adopted to fork the data from primary pipeline. This results in no disruption with the primary pipeline, at the same time ad-hoc demands can be fulfilled. 
+\item \textbf{Closed-loop Analytics}: Spring XD orchestrates the entire analytics loop - gathering data from any source, triggering actions, handling feedback loops from machine learning models, and computing real-time predictions.
+\item \textbf{Hadoop and Beyond}: Enterprises with existing investment in Hadoop technologies find Spring XD as one-stop orchestration platform. Whether it is MapReduce, Hive, Pig, or HBase scripts - developer experience is the same.
```

Comment:
'Spring XD, as one-stop runtime, provides data integration adapters to consume data of varied formats (i.e., structures, unstructured, binary, ..) that reside in different toolchains (i.e., database, middlewares, in-memory grids, ..). '   This statement left me hanging.  It consumes data and then what?